test/parallel/test-whatwg-url-searchparams-entries.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const URL = require('url').URL;
+
+const m = new URL('http://example.org');
+let params = m.searchParams;
+
+// Until we export URLSearchParams
+const URLSearchParams = params.constructor;
```

Comment:
This is also not needed any more after 326e967c6bddd1cfb901f7b12d06f9e3136243c1.